extensions/oidc-client-filter/deployment/src/main/java/io/quarkus/oidc/client/filter/deployment/OidcClientFilterBuildStep.java

```diff
@@ -24,15 +35,88 @@ public class OidcClientFilterBuildStep {
     @BuildStep
     void registerProvider(BuildProducer<AdditionalBeanBuildItem> additionalBeans,
             BuildProducer<ReflectiveClassBuildItem> reflectiveClass,
+            NamedOidcClientFilterBuildItem namedOidcClientFilterBuildItem,
             BuildProducer<ResteasyJaxrsProviderBuildItem> jaxrsProviders,
-            BuildProducer<RestClientAnnotationProviderBuildItem> restAnnotationProvider) {
+            BuildProducer<RestClientAdditionalProviderBuildItem> restAnnotationProvider) {
+
         additionalBeans.produce(AdditionalBeanBuildItem.unremovableOf(OidcClientRequestFilter.class));
         reflectiveClass.produce(new ReflectiveClassBuildItem(true, true, OidcClientRequestFilter.class));
+        final Set<String> namedFilterClientClasses = namedOidcClientFilterBuildItem.namedFilterClientClasses;
+
+        // register default request filter provider against the rest of the clients (client != namedFilterClientClasses)
```

Comment:
Please clarify this point in JavaDocs for `tegisterFilter`

Reply:
Added documentation and note, thx.